tests/manifest_generation_tutorial.md

```diff
@@ -0,0 +1,80 @@
+# About
+
+This document instructs how to generate `tests/goldens/manifest.json` for a particular dbt and manifest schema version.
+
+# Tutorial
+
+dbt version to Manifest version mapping is documented here: https://docs.getdbt.com/reference/artifacts/manifest-json
+
+
+## dbt Core v.1.5.4 (Manifest version v9)
+
+### 1. Prepare environment
+
+```
+virtualenv manifest_test --python=python3.9
+source manifest_test/bin/activate
+(manifest_test) % pip install -r requirements.txt
+```
+Where `requirements.txt` contains the following packages (please check which plugin version is compatible with a corresponding dbt version).
+
+```
+acryl-datahub[dbt]==0.10.4
+dbt-core==1.5.4
+dbt-spark==1.5.2
+dbt-bigquery==1.5.5
+dbt-postgres==1.5.4
+dbt-snowflake==1.5.2
+dbt-redshift==1.5.9
+```
+
+### 2. Init dbt
```

Comment:
Maybe we should create a separate repo with all the models required for generating this manifest? We wouldn't have to rely on `dbt init` command then (whose output is not in our control and hypothetically could be changed in the future).

Reply:
Indeed, `dbt init` output is beyond our control. I have such a project locally - we could deploy it to the repo on our GitLab/GitHub.

Reply:
Nevertheless, it'd be a bit surprising if this output changed given the exact version of dbt & manifest's schema.